packages/client/src/profile/Profile.ts

```diff
@@ -24,4 +28,30 @@ export class Profile {
 
     return result.data.result;
   }
+
+  async fetchAll(request: ProfileQueryRequest, observerId?: string) {
+    const result = await this.sdk.Profiles({
+      request,
+      observerId,
+    });
+
+    return result.data.result;
+  }
+
+  async recommendedProfiles(observerId?: string) {
+    const result = await this.sdk.RecommendedProfiles({
+      observerId,
+    });
+
+    return result.data.result;
+  }
+
+  async mutualFollowersProfiles(request: MutualFollowersProfilesQueryRequest, observerId?: string) {
```

Comment:
the method names in TODO are shorter to avoid repeating "profile"
```
- [Q] client.profile.fetch(request) (by id or handle, forSources) or client.profiles.fetchOne()
- [PQ] client.profile.fetchAll(request) (by ids, ownedBy, handles, whoMirroredPublicationId)
- [Q] client.profile.fetchRecommended()
- [PQ] client.profile.mutualFollowers({ viewingProfileId, yourProfileId })

```